src/llmcompressor/modifiers/awq/base.py

```diff
@@ -134,9 +136,14 @@ class AWQModifier(Modifier, QuantizationMixin):
 
     # Private vars set during initialization, cleared during finalization
     _resolved_mappings: List[ResolvedMapping] = PrivateAttr(default_factory=list)
-    _activations: Dict[str, List[torch.Tensor]] = PrivateAttr(default_factory=dict)
-    _activation_hooks: Set[RemovableHandle] = PrivateAttr(default_factory=set)
-    _module_kwargs: Dict = PrivateAttr(default_factory=dict)
+    # Cache list of forward input args for each parent module, one dict for each batch
+    _parent_args_cache: Dict[Module, IntermediatesCache] = PrivateAttr(
+        default_factory=dict
+    )
+    # Dict[smooth layer name, (activation means, activation counts)]
+    _smooth_activation_cache: Dict[str, Tuple[torch.FloatTensor, int]] = PrivateAttr(
```

Comment:
This is not really an activations cache, it doesn't store activations. Instead it stores means

Reply:
what would be a better name here? `_smooth_activation_means_cache`?

Reply:
renamed to `_smooth_activation_means`